scripts/update-codegen.sh

```diff
@@ -18,9 +18,9 @@ echo "Calling ${CODEGEN_PKG}/generate-groups.sh"
 
 echo "Generating other deepcopy funcs"
 "${GOPATH}"/bin/deepcopy-gen \
-  --input-dirs ./netmesh/model/netmesh \
+  --input-dirs ./netmesh/model/netmesh ./pkg/nsm/apis/common \
```

Comment:
the list syntax is not correct. It takes the second value as a normal command argument. You need another `--input-dirs` if I remember right. Same below.